src/view.js

```diff
@@ -30,8 +32,10 @@ export class View {
     publishNewAnswerEvent,
     publishPlayGameAgainEvent
   ) {
+    this.level;
     this.appContainer = musicApp;
     this.isPlayTonesButtonClicked = false;
+    this.hintButtonCounter = 0;
```

Comment:
Just a thought - do these need to be public fields? We've talked before about making fields/methods private by default, and only exposing them as public when necessary. This helps to keep the interfaces of our classes simpler, so it's eaiser to for other code to see what it does, and how to work with it. It also helps us limit ways that code outside of the class can modify instances of the class (e.g. setting the level directly). Could we do that here?

Reply:
All these don't need to be public actually. I'll update them now!